src/Storage/Storage.Management/ChangeLog.md

```diff
@@ -18,7 +18,7 @@
         - Additional information about change #1
 -->
 ## Upcoming Release
-* Updated Storage account cmdlet output context based on OAuth token 
```

Comment:
This change is in last Sep release, the change log is in wrong place, so move it to the right place.

Reply:
The change isn't included in the previous release https://github.com/Azure/azure-powershell/pull/25954. The change log should be here

Reply:
fixed.